test/unit/core/error_helpers.js

```diff
@@ -1,4 +1,94 @@
 suite('Error Helpers', function() {
+  var myp5 = new p5(function( sketch ) {
+    sketch.setup = function() {};
+    sketch.draw = function() {};
+  });
+  var c;
+
+  // unit tests for validateParameters
+  suite('validateParameters', function(){
+    test('documentation look up - arc()', function() {
+      assert.doesNotThrow(function() {
+          p5.prototype._validateParameters('arc',
+            [1, 1, 10.5, 10, 0, Math.PI, 'pie']);
+        },
+        Error, 'got unwanted exception');
+    });
+    test('documentation look up - arc()', function() {
+      assert.doesNotThrow(function() {
+          p5.prototype._validateParameters('arc',
+            [1, 1, 10.5, 10]);
+        },
+        Error, 'got unwanted exception');
+    });
+    test('documentation look up - arc()', function() {
+      assert.doesNotThrow(function() {
+          p5.prototype._validateParameters('arc',
+            ['1', 1, 10.5, 10, 0, Math.PI, 'pie']);
+        },
+        Error, 'got unwanted exception');
+    });
```

Comment:
This generates err msg:
`> p5.js says: FES: arc was expecting Number for parameter #0 (zero-based index), received "1" instead. [http://p5js.org/reference/#p5/arc]`